test/unit/app/presenters/publishing_api/editionable_worldwide_organisation_presenter_test.rb

```diff
@@ -39,10 +41,35 @@ def present(...)
       description: worldwide_org.summary,
       details: {
         body: "<div class=\"govspeak\"><p>Information about the organisation with <em>italics</em>.</p>\n</div>",
+        home_page_office_parts: [
+          {
+            access_and_opening_times: Whitehall::GovspeakRenderer.new.govspeak_to_html(worldwide_org.reload.home_page_offices.first.access_and_opening_times),
+            contact_content_id: worldwide_org.reload.home_page_offices.first.contact.content_id,
+            services: [],
+            slug: worldwide_org.reload.home_page_offices.first.slug,
```

Comment:
Just checking is this the full path `offices/office-1` or just `office-1`?

Reply:
I haven't looked at your Gov Frontend code yet, but in the spike I did the other week, this would have had to be `offices/office-1`

Reply:
It's `offices/office-1`.  I wasn't really sure whether to add the `offices/` here or in the frontend app.

I went with the frontend app in the end, as it isn't really part of the `slug`.

I wonder whether full base path (e.g. `/world/organisation/foo/offices/bar`) might actually be better than slug?

Reply:
I kind of see it relative to the content item it's part of... if that makes sense?

Reply:
Yeah, that's true.  Let's go for `office/office-1` here, then I can removed `office/` from the Government Frontend code.